app/uk/gov/hmrc/nisp/views/account_forecastonly.scala.html

```diff
@@ -0,0 +1,68 @@
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*@
+
+@import uk.gov.hmrc.nisp.utils.Constants
+@import uk.gov.hmrc.play.views.formatting.Dates
+@import uk.gov.hmrc.nisp.models.SPSummaryModel
+@import uk.gov.hmrc.nisp.controllers.routes
+@import uk.gov.hmrc.nisp.views.formatting.{NispMoney, Time}
+@import uk.gov.hmrc.nisp.services.ABService
+@import uk.gov.hmrc.nisp.models.enums.ABTest._
+
+
+@(nino: String, spSummary: SPSummaryModel, user: uk.gov.hmrc.nisp.controllers.auth.NispUser, abTest: Option[ABTest])(implicit request: Request[_])
+
+@analyticsAdditionalJs = {
+    ga('set', {
+                'dimension7':  '@spSummary.contextMessage.getOrElse("")',
+                'dimension8':  '@spSummary.forecastAmount.week.toInt',
+                @Html(abTest.map (p => s"'dimension9':  '${p.toString}',").getOrElse(""))
+                'dimension10': '@spSummary.numberOfQualifyingYears',
+                'dimension11': '@spSummary.numberOfGaps',
+                'dimension12': '@spSummary.numberOfGapsPayable',
+                'dimension13': '@spSummary.yearsToContributeUntilPensionAge',
+                'dimension14': '@spSummary.contractedOutFlag',
+                'dimension15': '@spSummary.statePensionAge',
+                'dimension16': '@spSummary.copeAmount.week',
+                'dimension21': '@spSummary.forecastOnlyFlag'
+    });
+}
+
+@nispMain(serviceInfoContent=Some(includes.loginInfo(user)), userLoggedIn = true, browserTitle = Messages("nisp.main.title"), analyticsAdditionalJs = Some(analyticsAdditionalJs)) {
+
+<div class="grid services">
+    <h1 class="heading-large title-spa">@Messages("nisp.main.h1.title")</h1>
+    <div class="highlighted-event">
+        <p id="amount-intro">@Messages("nisp.main.basedOn")<span style="white-space:nowrap"> @Dates.formatDate(spSummary.statePensionAge.date.localDate)
+             </span>@Messages("nisp.main.whenYoullBe") @spSummary.statePensionAge.age, @Messages("nisp.main.title.estimate.is")
+        </p>
+        <p>
+            <em id="amount-value">@NispMoney.pounds(spSummary.forecastAmount.week) @Messages("nisp.main.week")</em>
+            <p id="amount-sub">
+                 @NispMoney.pounds(spSummary.forecastAmount.month) @Messages("nisp.main.month"),
+                 @NispMoney.pounds(spSummary.forecastAmount.year) @Messages("nisp.main.year")
+            </p>
+        </p>
+    </div>
+    <div class="content-block">
+        <p>@Messages("nisp.main.guarantee.forecastonly", spSummary.lastProcessedDate.localDate.getYear().toString())</p>
+         @if(spSummary.hasPsod) {<p>@Messages("nisp.main.psod")</p>}
+    </div>
+    <h2 class="heading-medium" style="margin-top: 1.875em !important">@Messages("nisp.main.forecastonly.h2.title")</h2>
```

Comment:
Indeed it is @dspork . However there is an issue with the right margin-top not appearing because in assets frontend the h2 within an article has an !important tag which means the spacing is not correct.